src/components/home/Home.vue

```diff
@@ -1,13 +1,115 @@
 <template>
-    <div class="home">
-        <h1>{{ $t('home.title') }}</h1>
-        <p>{{ $t('home.content') }}</p>
+    <div class="home" v-bind:class="themes[themeIndex].themeName">
+        <b-navbar sticky>
+            <b-navbar-nav>
+                <b-nav-item v-for="(theme, index) in themes" :key="index" v-bind:activeTheme="theme.activeTheme"
+                            v-on:click="changeTheme(index)">
+                    <b-img height="50" v-bind:src="theme.imageSource"></b-img>
+                </b-nav-item>
+            </b-navbar-nav>
+        </b-navbar>
+        <b-container>
+            <b-row>
+                <b-col cols="2"></b-col>
+                <b-col cols="8">
+                    <chat-box v-bind:bot-icon-source="themes[themeIndex].imageSource"></chat-box>
+                </b-col>
+                <b-col cols="2">
+                    <b-img thumbnail center width="100%" class="logoImage"
+                           :src="require('../../assets/botLogo.png')"></b-img>
+                </b-col>
+            </b-row>
+        </b-container>
     </div>
 </template>
 <script>
+    import ChatBox from "../ChatBox/ChatBox";
+
     export default {
         name: 'Home',
+        components: {ChatBox},
+        data: function () {
+            return {
+                themeIndex: 0,
+                themes: []
+            }
+        },
+        methods: {
+            changeTheme: function (index) {
+                this.themeIndex = index;
+                for (let theme of this.themes) {
+                    theme.activeTheme = false;
+                }
+                this.themes[index].activeTheme = true;
+            }
+        },
+        created() {
+            this.themes.push({
```

Comment:
Podobnie, jak poprzednio zapełnijmy tą tablicę od razu podczas stworzenia. Jeżeli się da, to ściągnijmy te obrazki i trzymajmy w assetsach